dataretrieval/wqp.py

```diff
@@ -468,3 +486,19 @@ def _alter_kwargs(kwargs):
     kwargs['mimeType'] = 'csv'
 
     return kwargs
+
+def _three_point_oh_warn():
+    """Private function for warning message about WQX 3.0 profiles
+    """
+
+    warnings.warn('NEWS: USGS data availability and format are changing. '\
```

Comment:
This is pretty verbose. I would just warn that new USGS data will be unavailable for several weeks during the transition to new WQX profiles and refer them to a URL.
No need to add a backslash to the end of the line, so long as your strings are bracketed.
```python
example = (
    'This is a test. '
    'This line will continue the previous one.'
)
```

Reply:
@ehinman this is the shortened message you can use for the WQP functions: 
NEWS: USGS data availability and format are changing. 
Beginning in February 2024 the data obtained from legacy profiles
will not include new USGS data or recent updates to existing data. 
To view the status of changes in data availability and code functionality, visit:
https://doi-usgs.github.io/dataRetrieval/articles/Status.html
If you have additional questions about these changes, 
email CompTools@usgs.gov.

Reply:
This looks good. Consider adding this warning to the README and note the expected outage in February. A lot of users will be automated scripts or people with older packages, so they likely won't see this in time, but the README might be one of the first places they check.